sdk/nodejs/src/request.ts

```diff
@@ -1,11 +1,23 @@
 import axios, { AxiosRequestConfig, AxiosRequestHeaders, AxiosResponse } from 'axios'
+import axiosRetry from 'axios-retry'
 import { SDKEventBatchRequestBody, DVCLogger } from '@devcycle/types'
 import { DVCPopulatedUser } from './models/populatedUser'
 import { DVCEvent, DVCOptions } from './types'
 
 const axiosClient = axios.create({
+    timeout: 10 * 1000,
     validateStatus: (status: number) => status < 400 && status >= 200,
 })
+axiosRetry(axiosClient, {
+    retries: 5,
```

Comment:
the problem now is that the node SDK is still refetching the config every 10 seconds from the environmentConfigManager, so if theres a lot of request failures happening this will now pile up an increasing set of retrying requests

Reply:
so one option would be to make the config fetch timeout be 1 sec, and I think 5 retries would still then take less than 5 sec to complete. But the problem is that this is kinda a global config that applies to all requests...

Reply:
- Setup multiple axios clients for retryable requests, and non-retryable requests (events)
- Only log error if config fetch errors on first fetch.

Reply:
won't this still be a problem with the latest changes since you'll still be retrying a timed out request up to 5 times while still proceeding with the 10 second config fetching loop

Reply:
added a new axios client for config requests that will do a single retry.